Experiments_toolkit.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+from skimage.util.shape import view_as_windows
+import windowing as w
+
+
+def create_train_test_validation(dictionary, pos_key, neg_key, percent_train, percent_test, percent_validation):
+    """
+        Description: creates a training, testing, and validation dictionary for binary data
+
+        Args:
+            dictionary -- dict (Dictionary of data sorted into pos_key and neg_key:
```

Comment:
after our discussion today, I assume you'll convert all these functions to use a numpy array?

Reply:
Yep!